packages/framework/pr_tools/SetKokkosDevelop.sh

```diff
@@ -18,3 +18,15 @@ message_std "SetKokkosDevelop> INFO: updated kokkos and kokkos-kernels packages
 
 # Returns to previous path from before running this script
 cd -
+
+ln -s "$PACKAGESPATH/kokkos" "${WORKSPACE:?}/Trilinos/packages/kokkos"
+
+cmake ${WORKSPACE:?}/Trilinos -DKokkos_SOURCE_DIR_OVERRIDE:STRING=kokkos
```

Comment:
This may work, though I do get a bit concerned about running cmake before we're running it "for real" (if we run it the second time it may be okay).  A better way would be to add this to the `EXTRA_CONFIGURE_FLAGS` in the script up higher, and pass it to this script as the place to check out kokkos/kokkos-kernels and then do the symlinking.